integration/commands_administration_test.go

```diff
@@ -682,7 +682,7 @@ func TestCommandsAdministrationDataSize(t *testing.T) {
 		assert.Equal(t, float64(1), must.NotFail(doc.Get("ok")))
 		assert.InDelta(t, float64(16_384), must.NotFail(doc.Get("size")), 16_384)
 		assert.InDelta(t, float64(4), must.NotFail(doc.Get("numObjects")), 4) // TODO https://github.com/FerretDB/FerretDB/issues/727
-		assert.InDelta(t, float64(100), must.NotFail(doc.Get("millis")), 100)
```

Comment:
Shouldn't that be in seperate PR?

Reply:
As the tests started failing in this PR, I believe it should be in this PR :)
I guess that those tests started failing because there is more data in this PR (all the extra collections and values for nil), so 100 is not enough anymore.